apps/specter-desktop/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ x-logging:
 
 services:
   web:
-    image: lncm/specter-desktop:v1.2.2@sha256:22d6abe4ba97dcb9d3f5f1695524c5262be176d873f5f57c9d9c3a95840cb3da
+    image: lncm/specter-desktop:v1.3.1@sha256:c72fc1f270d6e5f8426f185e5c08fb8581728ad5c6ec943d719a9d08ce996c15
```

Comment:
Can I ask where you got the sha256 digest for this? I was trying to open a PR for this myself, and I couldn't locate where the one for the previous version came from, and so I couldn't be sure I was getting the correct one for the updated version. And I am not able to locate this specific digest you have listed here.

I'm looking at this page: https://hub.docker.com/layers/lncm/specter-desktop/v1.3.1/images/sha256-c24068af20f6a3fde96be9128827fde1e638db32c7bfde0e0f53e1be063caffb

None of the architectures on that page have the SHA you posted, which makes me think I'm looking in the wrong place.

Reply:
I've just `docker pull`ed it, which displayed a sha256 that works for all architectures.